wgsl/extract-grammar.py

```diff
@@ -419,6 +425,78 @@ def not_token_only(value):
     grammar_package.write('    "main": "bindings/node"\n')
     grammar_package.write('}\n')
 
+# External scanner for nested block comments
+# See: https://github.com/tree-sitter/tree-sitter-rust/blob/master/src/scanner.c
+
```

Comment:
Nitpick:  I suggest also referencing https://tree-sitter.github.io/tree-sitter/creating-parsers#external-scanners
That explains the API in detail.
(Except it doesn't explain the 'reset' method.  oops. :-)  )

Reply:
@dneto0 definitely makes sense, tried here: https://github.com/gpuweb/gpuweb/pull/2292

Reply:
Interesting.  The docs are more up to date than the Rust scanner.
The reset method can be removed.

Reply:
@dneto0 indeed that builds fine, should have checked in the first place...: https://github.com/gpuweb/gpuweb/pull/2293